test/support/fixtures/invocation_fixtures.ex

```diff
@@ -29,7 +32,8 @@ defmodule Lightning.InvocationFixtures do
         Lightning.ProjectsFixtures.project_fixture().id
       end)
       |> Keyword.put_new_lazy(:dataclip_id, fn ->
-        dataclip_fixture().id
+        # TODO: How do we only pass certain attrs here? Just the project_id
+        dataclip_fixture(attrs).id
```

Comment:
In this case I would split out the creation of the `project_id`.

So `attrs = attrs |> Keyword.put_new_lazy(:project_id, ...)` and then in the subsequent put new lazys take the `project_id` out of the attrs with `dataclip_fixture(project_id: Keyword.get(:project_id)`